src/Process/3dTilesProcessing.js

```diff
@@ -26,6 +27,32 @@ function subdivideNode(context, layer, node, cullingTest) {
     }
 }
 
+function boundingVolumeToExtent(crs, layer, volume, transform) {
+    if (volume.region) {
+        return new Extent('EPSG:4326',
+            THREE.Math.radToDeg(volume.region[0]),
+            THREE.Math.radToDeg(volume.region[2]),
+            THREE.Math.radToDeg(volume.region[1]),
+            THREE.Math.radToDeg(volume.region[3]));
+    } else if (volume.box) {
+        const box = volume.box.clone().applyMatrix4(transform);
+        return new Extent(crs, {
+            west: box.min.x,
+            east: box.max.x,
+            south: box.min.y,
+            north: box.max.y,
+        });
+    } else {
+        const sphere = volume.sphere.clone().applyMatrix4(transform);
+        return new Extent(crs, {
+            west: sphere.center.x - sphere.radius,
+            east: sphere.center.x + sphere.radius,
+            south: sphere.center.y - sphere.radius,
+            north: sphere.center.y + sphere.radius,
```

Comment:
is transform restricted to affine transforms with isotropic scaling or is applyMatrix4 ensuring a bounding sphere of the resulting quadric ?

Reply:
The scaling is not necessarly isotropic but, yes, three.js scales the radius by the largest scale factor (see https://github.com/mrdoob/three.js/blob/master/src/math/Sphere.js#L157) so we're good.

Reply:
Do we really need a comment saying "Works as intended" ?